packages/cli/src/commands/setup/middleware/ogImage/ogImageHandler.ts

```diff
@@ -102,8 +102,8 @@ export async function handler({ force }: { force: boolean }) {
             "og:image generation is almost ready to go! You'll need to add playwright as a dependency to the api side and then install the headless browser packages:",
```

Comment:
This sentence mentions `api` as well!

```suggestion
            "og:image generation is almost ready to go! You'll need to add playwright as a dependency to the web side and then install the headless browser packages:",
```